src/Command/AddUserCommand.php

```diff
@@ -44,6 +44,10 @@
  */
 class AddUserCommand extends Command
 {
+    // to make your command lazily loaded, configures the $defaultName static property,
```

Comment:
Typo: `configures the $defaultName...` -> `configure the $defaultName...`

Suggestion: `only when the app:add-user command is actually called` -> `only when the command is actually called`

Reply:
Thanks Javier, done.